components/html-block/html-block.js

```diff
@@ -150,8 +187,9 @@ class HtmlBlock extends LitElement {
 				let temp = document.createElement('div');
 				temp.appendChild(fragment);
 
+				const context = await getContext();
 				for (const render of HtmlBlock._renderers) {
-					temp = await render(temp);
+					temp = await render(temp, context);
```

Comment:
Hmmmm. I think it would be preferred if the renderer can get the context themselves, as opposed to depending on html-block passing in an object that contains everything it needs.  Was this something that was considered?

Reply:
Yeah, I didn't have any strong reason for doing it this way. It felt slightly cleaner to centralize things, but I figured you would have ideas on this when you got back, since I don't really know the long-term vision for this. I can easily make this change. :)